src/styles/app/35-sidebar-nav.css

```
@@ -139,6 +135,15 @@
 	fill: currentColor;
 }
 
+/* ── New note: filled accent icon ── */
+.sidebarNavBtn[data-kind="new-note"] > svg {
+	color: var(--interactive-accent);
+}
+
+.sidebarNavBtn[data-kind="new-note"] > svg path {
+	fill: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)